doc/src/conf.py

```diff
@@ -1,3 +1,26 @@
+###########################################
+#  The MIT License (MIT)
+#  
+#  Copyright (c) 2015-2022 Advanced Micro Devices, Inc. All rights reserved.
+#  
+#  Permission is hereby granted, free of charge, to any person obtaining a copy
+#  of this software and associated documentation files (the "Software"), to deal
+#  in the Software without restriction, including without limitation the rights
+#  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#  copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#  
+#  The above copyright notice and this permission notice shall be included in
+#  all copies or substantial portions of the Software.
+#  
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#  THE SOFTWARE.
+###########################################
```

Comment:
Looks like this file is auto-generated.  I am not sure if we should add a license on this or not.

Reply:
How do we want to handle auto gen'd files then? This sounds liek I should have another file with a list of thing not to stamp then

Reply:
Its not completely autogenerated. It was initially generated but then we modified with out settings, It should still have a license.

Reply:
What about rules for generated files then? Can we assume since "we" generated it, we should stamp it with our license?